FirmataParser.cpp

```diff
@@ -163,8 +166,8 @@ void FirmataParser::parse(uint8_t inputData)
         systemReset();
         break;
       case REPORT_VERSION:
-        if (currentReportVersionCallback)
-          (*currentReportVersionCallback)(currentReportVersionCallbackContext);
+        waitForData = 2; // two data bytes needed
```

Comment:
Actually `START_SYSEX` will get through because `( (waitForData > 0) && (inputData < 128) )`
But I don't think the COMMAND byte will because `waitForData` would still equal 2. One work around is in the START_SYSEX case, reset waitForData to 0.